bot/cogs/doc.py

```diff
@@ -150,16 +175,37 @@ def __init__(self, bot: commands.Bot):
         """
         self.base_urls[package_name] = base_url
 
-        fetch_func = functools.partial(intersphinx.fetch_inventory, config, '', inventory_url)
-        for _, value in (await self.bot.loop.run_in_executor(None, fetch_func)).items():
-            # Each value has a bunch of information in the form
-            # `(package_name, version, relative_url, ???)`, and we only
-            # need the relative documentation URL.
-            for symbol, (_, _, relative_doc_url, _) in value.items():
-                absolute_doc_url = base_url + relative_doc_url
-                self.inventories[symbol] = absolute_doc_url
-
-        log.trace(f"Fetched inventory for {package_name}.")
+        package = await self._fetch_inventory(inventory_url, config)
+        if package:
+            for group, value in package.items():
+                # Each value has a bunch of information in the form
+                # `(package_name, version, relative_url, ???)`, and we only
+                # need the package_name and the relative documentation URL.
+                for symbol, (package_name, _, relative_doc_url, _) in value.items():
+                    absolute_doc_url = base_url + relative_doc_url
+
+                    if symbol in self.inventories:
+                        # get `group_name` from _:group_name
+                        group_name = group.split(":")[1]
+                        if (group_name in NO_OVERRIDE_GROUPS
+                                # check if any package from `NO_OVERRIDE_PACKAGES`
+                                # is in base URL of the symbol that would be overridden
+                                or any(package in self.inventories[symbol].split("/", 3)[2]
+                                       for package in NO_OVERRIDE_PACKAGES)):
+
+                            symbol = f"{group_name}.{symbol}"
+                            # If renamed `symbol` already exists, add library name in front.
+                            if symbol in self.renamed_symbols:
```

Comment:
The comment is describing the code again here, but not explaining the why. The comment would be more useful if it describes why it's adding the library name in front such as "for disambiguation" or such.